QRScanner/QRScannerView.swift

```diff
@@ -241,23 +270,25 @@ public class QRScannerView: UIView {
             strongSelf.qrCodeImageView.frame = path.bounds
             strongSelf.qrCodeImageView.center = center
             }, completion: { [weak self] _ in
+                self?.qrCodeImageView.image = self?.qrCodeImage
                 self?.delegate?.success(qrCode)
         })
     }
 
-    private func displayQRCode(_ image: UIImage) {
-        DispatchQueue.main.asyncAfter(deadline: .now() + animationDuration) { [weak self] in
```

Comment:
This is not sync to square rect animation completion https://github.com/mercari/QRScanner/pull/3/files#diff-9fb699fc4f6cb45bb232a516c714e613R273

This is executed after `delegate.success(qrCode)` calling.
To sync them, keeps current image to instance property. https://github.com/mercari/QRScanner/pull/3/files#diff-9fb699fc4f6cb45bb232a516c714e613R321